macros/nsprapiref.ejs

```diff
@@ -11,497 +11,553 @@ function state(section) {
-      'Linked Lists' : 'Linked Lists',
-      'Linked List Types' : 'Linked List Types',
-      'Linked List Macros' : 'Linked List Macros',
-      'Dynamic Library Linking' : 'Dynamic Library Linking',
-      'Library Linking Types' : 'Library Linking Types',
-      'Library Linking Functions' : 'Library Linking Functions',
-      'Finding Symbols Defined in the Main Executable Program' : 'Finding Symbols Defined in the Main Executable Program',
-      'Platform Notes' : 'Platform Notes',
-      'Dynamic Library Search Path' : 'Dynamic Library Search Path',
-      'Exporting Symbols from the Main Executable Program' : 'Exporting Symbols from the Main Executable Program',
-      'Process Management and Interprocess Communication' : 'Process Management and Interprocess Communication',
-      'Process Management Types and Constants' : 'Process Management Types and Constants',
-      'Process Management Functions' : 'Process Management Functions',
-      'Setting the Attributes of a New Process' : 'Setting the Attributes of a New Process',
-      'Creating and Managing Processes' : 'Creating and Managing Processes',
-      'Multiwait Receive' : 'Multiwait Receive',
-      'System Information and Environment Variables' : 'System Information and Environment Variables',
-      'Logging' : 'Logging',
-      'Conditional Compilation and Execution' : 'Conditional Compilation and Execution',
-      'Log Types and Variables' : 'Log Types and Variables',
-      'Logging Functions and Macros' : 'Logging Functions and Macros',
-      'Use Example' : 'Use Example',
-      'Instrumentation Counters' : 'Instrumentation Counters',
-      'Named Shared Memory' : 'Named Shared Memory',
-      'Shared Memory Protocol' : 'Shared Memory Protocol',
-      'Named Shared Memory Functions' : 'Named Shared Memory Functions',
-      'Anonymous Shared Memory' : 'Anonymous Shared Memory',
-      'Anonymous Memory Protocol' : 'Anonymous Memory Protocol',
-      'Anonymous Shared Memory Functions' : 'Anonymous Shared Memory Functions',
-      'IPC Semaphores' : 'IPC Semaphores',
-      'IPC Semaphore Functions' : 'IPC Semaphore Functions',
-      'Thread Pools' : 'Thread Pools',
-      'Thread Pool Types' : 'Thread Pool Types',
-      'Thread Pool Functions' : 'Thread Pool Functions',
-      'Random Number Generator' : 'Random Number Generator',
-      'Random Number Generator Function' : 'Random Number Generator Function',
-      'Hash Tables' : 'Hash Tables',
-      'Hash Tables and Type Constants' : 'Hash Tables and Type Constants',
-      'Hash Table Functions' : 'Hash Table Functions',
-      'NSPR Error Handling' : 'NSPR Error Handling',
-      'Error Type' : 'Error Type',
-      'Error Functions' : 'Error Functions',
-      'Error Codes' : 'Error Codes',
-      'Contribute' : 'Contribute',
-      'Doc_Status' : 'NSPR Documentation status',
-      'The_MDN_project' : 'The MDN Project'
-
+    'NSPR' : 'NSPR',
```

Comment:
``{{nsprapiref}}`` is a big one, and it converted correctly 👍, as seen on https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR/Reference/NSPR_Types:

<img width="496" alt="pr 789 nsprapiref" src="https://user-images.githubusercontent.com/286017/45397177-6aa32000-b604-11e8-8616-57122b810b14.png">